apps/sms/services/js/threads.js

```diff
@@ -164,5 +164,18 @@
     }
   };
 
+  Threads.Messages = {
+    get(id) {
+      id = +id;
+      var threadId = messageMap.get(id);
+
+      if (Threads.has(threadId)) {
+        return Threads.get(threadId).messages.get(id);
+      }
+
+      return null;
+    }
```

Comment:
__note:__ Right now I have mixed feelings, theoretically "client" who knows message id should know thread id as well and use ```Threads.get(threadId).getMessage(messageId) or Thread.getMessage(id)``` and not rely on sort of magic behind the scene... But I haven't seen usages yet to have full-fledged opinion :)

Also I've spent some time to understand why ```messageMap``` returns thread id by message id, imo it should be named more like ```messageIdToThreadIdMap``` :D